handlers/error.go

```diff
@@ -187,3 +190,15 @@ func ErrApplyChange(err error) error {
 func ErrRetrieveMeshData(err error) error {
 	return errors.New(ErrRetrieveMeshDataCode, errors.Alert, []string{"Error getting operations for the mesh", "Error getting service mesh name"}, []string{err.Error()}, []string{"unable to retrieve the requested data"}, []string{})
 }
+
+func ErrApplicationFailure(err error, obj string) error {
+	return errors.New(ErrApplicationFailureCode, errors.Alert, []string{"failed to ", obj, "the application"}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrDecoding(err error, obj string) error {
+	return errors.New(ErrDecodingCode, errors.Alert, []string{"Error decoding the : ", obj}, []string{err.Error()}, []string{"Unable to decode the : ", obj}, []string{})
```

Comment:
@sayantan1413 Uve filled `"Unable to decode the : "` in probable cause, lets have It blank.